algo.js

```diff
@@ -134,7 +140,7 @@ const checkMyersBriggs = (match, suitor, legend, myersBriggs, ranking) => {
   }
 }
 
-async function main(prevDocs) {
+async function main() {
```

Comment:
maybe give the `main()` function a more meaningful name... like `generateMatches()` or something like that